.buildkite/publish.sh

```diff
@@ -31,7 +31,7 @@ buildkite-agent artifact download "_out_/**/*" .
 echo "--- releasing sorbet.run"
 
 rm -rf sorbet.run
-git clone git@github.com:sorbet/sorbet.run.git
+git clone --single-branch git@github.com:sorbet/sorbet.run.git
```

Comment:
I thought that we should use shallow clones for this, and

https://www.perforce.com/blog/vcs/git-beyond-basics-using-shallow-clones

was the first thing that popped up, which says that `--single-branch` is basically the `--depth` thing I was thinking of.  It suggests using `git clone $URL --branch $BRANCH --single-branch` instead.  Is the `master` implicit here and/or is it worth making explicit?

Reply:
The `master` is implicit here. I'll go ahead and explicify it.